mne/io/tests/test_pick.py

```diff
@@ -94,9 +94,12 @@ def test_pick_refs():
         if len(pick) > 0:
             pick_info(info, pick)
 
+    msg = 'Removing %d compensators' % len(info['comps'])
     for pick in (picks_meg, picks_mag):
         if len(pick) > 0:
-            pytest.raises(RuntimeError, pick_info, info, pick)
+            with catch_logging() as log:
+                pick_info(info, pick)
```

Comment:
you need `verbose=True` here because this is a `info`-level warning. This fails on my machine because I have `MNE_LOGGING_LEVEL=warning` in my config

Reply:
(and you'll need to add `verbose` kwarg and decorator to `pick_info`, too)